zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/config/KeyManagers.scala

```diff
@@ -13,7 +13,7 @@ import javax.net.ssl.{ KeyManager, KeyManagerFactory }
 
 private object KeyManagers {
 
-  private def getDefaultKeyStore: ZIO[System, Throwable, KeyStore] =
+  private def getDefaultKeyStore: ZIO[Scope, Throwable, KeyStore] =
```

Comment:
There's no need to extend the managed scope of this outside of `getDefaultKeyStore`. The old version was `ZIO`, meaning that it handled the opening and closing of the key file inside as a black box, not leaking it through its interface. Let's keep it that way.

Reply:
Agreed, I think find/replace got the better of me.

Reply:
Done.